packages/markers/src/browser/problem/problem-widget.tsx

```diff
@@ -32,7 +34,8 @@ export class ProblemWidget extends TreeWidget {
         @inject(ProblemManager) protected readonly problemManager: ProblemManager,
         @inject(TreeProps) readonly treeProps: TreeProps,
         @inject(ProblemTreeModel) readonly model: ProblemTreeModel,
-        @inject(ContextMenuRenderer) readonly contextMenuRenderer: ContextMenuRenderer
+        @inject(ContextMenuRenderer) readonly contextMenuRenderer: ContextMenuRenderer,
+        @inject(SelectionService) readonly selectionService: SelectionService
```

Comment:
For new `injections` please use `property injections` as new `constructor` injections are a [breaking change](https://github.com/theia-ide/theia/wiki/Coding-Guidelines#property-injection).